src/api.ts

```diff
@@ -59,6 +59,13 @@ export function onConnected(callback: (provider: WebLNProvider) => void) {
   };
 }
 
+/**
+ * @returns the configuration of the current connector (if connected)
+ */
+export function getConnectorConfig() {
```

Comment:
do people need to know about the `connector` word here? 

can it be `getConfig`?

(even though somehow I pesonally like an API like `BitcinConnect.config`somewhat better BitcinConnect.isConnected)

Reply:
There is also the Bitcoin Connect config (the app name, filters, showBalance flag etc.). I am not sure it is good to be called `getConfig` or `config`

Connectors are mentioned often in the README right now.

Reply:
The API is not an object so we can't use a getter currently. I created a separate issue for that and tag it for `v4`: https://github.com/getAlby/bitcoin-connect/issues/187

Reply:
can we return one config object? 
trying to avoid that we come tomorrow and need another function for the other config.

Reply:
I don't think there is any need to ask for that, since the developer already specified it when initializing the library. It is also difficult to do with the current setup. I think it would be better to implement this as part of #187 when that is done.

Reply:
what's the reason to limit it? afaik the config is well defined and what is the reason to only return parts of it? 

hmm. I am not sure about that. in your POS app I already had the situation.